src/util/config.js

```diff
@@ -1,97 +1,168 @@
-import Joi from 'joi';
+import fs from 'fs';
```

Comment:
We should probably have unit tests for this. I'm wondering what we can test? That it's exports a valid JSON object? Test for specific config params?